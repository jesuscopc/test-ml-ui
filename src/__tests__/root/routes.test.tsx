import React from 'react';
import { shallow } from 'enzyme';
import Routes from 'root/routes';

describe('Suit test Routes', () => {
  let wrapper = shallow(<Routes />);
  beforeEach(() => {
    wrapper = shallow(<Routes />);
  });

  test('should be render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
