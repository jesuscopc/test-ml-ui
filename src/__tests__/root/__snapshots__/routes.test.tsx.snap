// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Suit test Routes should be render correctly 1`] = `
<Switch>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/items"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/items/:productId"
  />
</Switch>
`;
